metadatahandler.py

```diff
@@ -1,16 +1,80 @@
+import collections
+from enum import Enum
 import os
+import platform
+import util
 import shutil
-import collections
 import sys
-import platform
 import xml.etree.ElementTree as etree
 from xml.dom import minidom
-import util
 
 DosGame = collections.namedtuple('DosGame',
                                  'dosname metadataname name genres publisher developer year frontPic manual desc')
 
 
+class GameGenres(Enum):
```

Comment:
Thanks for The PR!
Did you do a comparison of the before and after generated genres for the games ?

For the name of this enum, I think simply Genre in the singular will be better.

Reply:
Done, see second commit

Reply:
Well you're gonne hate me but I feel it's now much more complicated than your first version and even more than my initial version...
The alphabetical order wasn't working ? Because I wasn't opposed to it per se

Reply:
I found that "Action" and "Arcade" almost always took precedence, so I had to add logic to prevent that.

Other than that, there were some categorizations that changed, that were not necessarily bad, but did not match the original. 

For example, Turrican was categorized as Shmup, and if I don't put an exception it comes up as "Platform" (arguably it's a bit of both)

I tried to simplify the code, hope this is more palatable. The alphabetical lookup now runs for many games but not all...